flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSource.java

```diff
@@ -45,7 +45,9 @@ public MySqlDataSource(MySqlSourceConfigFactory configFactory) {
     public EventSourceProvider getEventSourceProvider() {
         MySqlEventDeserializer deserializer =
                 new MySqlEventDeserializer(
-                        DebeziumChangelogMode.ALL, sourceConfig.isIncludeSchemaChanges());
+                        DebeziumChangelogMode.ALL,
+                        sourceConfig.isIncludeSchemaChanges(),
+                        sourceConfig.getJdbcProperties());
```

Comment:
I'm thinking if we need to pass the whole `JdbcProperties` into the deserializer, since `tinyInt1isBit` is the only key that relevant to type deserialization. Maybe passing a `boolean tinyInt1isBit` is enough?

Reply:
+ 1